packages/opentelemetry-resources/src/detectors/BrowserDetector.ts

```diff
@@ -31,7 +31,8 @@ class BrowserDetector implements Detector {
     const browserResource: ResourceAttributes = {
       [SemanticResourceAttributes.PROCESS_RUNTIME_NAME]: 'browser',
       [SemanticResourceAttributes.PROCESS_RUNTIME_DESCRIPTION]: 'Web Browser',
-      [SemanticResourceAttributes.PROCESS_RUNTIME_VERSION]: navigator.userAgent
+      [SemanticResourceAttributes.PROCESS_RUNTIME_VERSION]: navigator.userAgent,
+      ...getBrowserAttributes()
```

Comment:
@martinkuba can you review this - the `process.runtime.*` attributes were here before you added the `browser.*` attributes in the semantic conventions - do we really need process.runtime.* attributes as part of `BrowserDetector`?

cc: @MSNev @t2t2

Reply:
I believe removing the existing attributes from the detector can be breaking. As the `browser.*` attributes are the newer ones, is it possible to remove the duplicated entry `browser.user_agent`?